test/test_parser_ntriplesstar.py

```diff
@@ -0,0 +1,96 @@
+
+import pytest
+
+from pathlib import Path
+from shutil import copyfile
+from tempfile import TemporaryDirectory
+
+from rdflib.exceptions import ParserError
+
+from rdflib import Graph
+from rdflib.util import guess_format
+
+
+from rdflib.plugin import register
+from rdflib.parser import Parser
+
+register(
+    "ntstar",
+    Parser,
+    "rdflib.plugins.parsers.ntriples-star",
+    "NtriplesStarParser",
+)
```

Comment:
Is this (and similar in other parser tests) needed here if you are already have

https://github.com/RDFLib/rdflib/blob/e0d5746b26d43c9e476b995ee14304173206329f/rdflib/plugin.py#L267-L284